### DISCUSSION TITLE
Getting periodic slack notifications for disabled auto-sync feature in ArgoCD

### DISCUSSION THREAD

**Discussion by rishavjayswal** [argoproj/argo-cd#9343] [Ideas]
We recently had an issue with one of our releases and had to rollback all our services manually. While doing so we had to disable the ‘auto-sync’ feature. After reverting the faulty PR, we forgot to enable the auto-sync again and the apps were out-of-sync for a day. Is there a way we can enable a trigger or an alert that gets triggered after every duration (say one hour) and notify us on slack that the autosync feature is disabled?

We would also like to enable the notification after every PR merge, something like "The commit can not be released because auto-sync is disabled.".

**Comment by tamsky** [comment]
This may help someone:

```jsonnet
            alert: 'ArgoCdAppAutoSyncDisabled',
            expr: |||
              sum(
                argocd_app_info{
                  %(argoCdSelector)s,
                  autosync_enabled!="true",
                  name!~"%(argoAutoSyncDisabledIgnoredApps)s"
                }
              ) by (job, dest_server, project, name, autosync_enabled)
              > 0
```

https://github.com/adinhodovic/argo-cd-mixin/blob/3733302a12ec49d71b10f9e094fb1ad76589d8e1/alerts/alerts.libsonnet#L50-L60
